### ISSUE TITLE
Thexder: Arrow keys do not work with machine=pcjr

### ISSUE THREAD

**Issue by weirddan455** [dosbox-staging/dosbox-staging#3461]
### Are you using the latest Dosbox-Staging Version?

- [X] I have checked releases and am using the latest release.

### What Operating System are you using?

Linux x86_64

### If Other OS, please describe

Arch Linux

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

Neither the arrow keys nor the numpad works to move the character.  I have tried turning numlock off.  With `machine = tandy` the input works fine.

### Steps to reproduce the behaviour.

Explain how to reproduce
1.  Set `machine = pcjr` in the config
2.  Launch the game.  The enter key works to go past the intro key.
3.  Press the arrow keys.
4.  Character does not move.

### Download URL of affected game or software

https://www.myabandonware.com/game/thexder-dk

### Your configuration

```ini
[dosbox]
machine = pcjr
```

### Code of Conduct & Contributing Guidelines

- [X] Yes, I agree.

**Comment by johnnovak** [comment]
Does it work with previous releases, e.g., 0.80.1? How about DOSBox SVN or X?

**Comment by weirddan455** [comment]
Broken on v0.80.1 and Dosbox-X.  I don't have a build of SVN handy.

**Comment by weirddan455** [comment]
Maybe this isn't a bug but the control bindings are very weird compared to tandy if so.  These keys work to control the player:

Up - [
Left - L
Right - '
Down - .

I don't know what the hell kind of weird controls those are but there you go.

**Comment by Jkapp76** [comment]
I've seen an old booter game with this same issue. It might even be frogger. Not sure how it plays on read hardware. It'd be nice to also use the arrow keys on these if that wouldn't create a problem somewhere else.

**Comment by weirddan455** [comment]
>  I've seen an old booter game with this same issue. It might even be frogger. Not sure how it plays on real hardware. It'd be nice to also use the arrow keys on these if that wouldn't create a problem somewhere else.

You can use the mapper to re-map bindings.  Start dosbox with `--startmapper`.  I think there's also a hotkey to launch the mapper.

**Comment by johnnovak** [comment]
> I think there's also a hotkey to launch the mapper.

Ctrl + F1 (Cmd + F1 on macOS).

**Comment by johnnovak** [comment]
So can this be closed then @weirddan455 ? It's just how the game behaves on PCjr?

**Comment by weirddan455** [comment]
Yeah, not a bug.  I just found a copy of the manual where it documents these weird ass controls for pcjr:  https://www.mocagh.org/sierra/thexder-manual.pdf

**Comment by johnnovak** [comment]
> Yeah, not a bug. I just found a copy of the manual where it documents these weird ass controls for pcjr: https://www.mocagh.org/sierra/thexder-manual.pdf

Glad you were able to confirm this 👍🏻
